recipes/parambokeh/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.2.3" %}
+
+package:
+  name: parambokeh
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/p/parambokeh/parambokeh-{{ version }}.tar.gz
+  sha256: 1b406581b85df5d955de8e744fce969b67e36e95ea3e3c496ec8e4e72029feaf
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - pyct-core >=0.4.4
+  run:
+    - python
+    - param >=1.7.0
+    - bokeh >=0.12.10
```

Comment:
I probably would have expected to see pyviz_comms here, but I guess it depends how you want to use parambokeh.